scripts/postgres_interface.py

```diff
@@ -15,7 +15,7 @@ def get_history_data():
 
     cursor = conn.cursor()
 
-    cursor.execute("select file_names from history")
+    cursor.execute("select file_names from history where success")
```

Comment:
mainly curious (and feel free to disregard): what's the significance of the `success` field?

Reply:
History table will be updated even the report uploading failed. I included this to allows retry on report that failed to upload previously